### DISCUSSION TITLE
MySQL

### DISCUSSION THREAD

**Discussion by silvasbr** [AzuraCast/AzuraCast#5060] [General]
Hi.
Is it possible to set up a music database in My SQL, and use it in azuracast?

**Comment by ivellios1988** [comment]
AzuraCast already uses a database (MariaDB, which is a fork of MySQL). Why do you want to create a separate one?

**Comment by silvasbr** [comment]
Hi.

every time I change servers. I need to upload all the songs again.

I already tried to backup. but it gives error.

I just wanted to know if it's possible to create a database with just the
songs. and apply that database on the new server.

so you don't have to keep uploading all the songs again.

the backup without the media worked. I managed to apply it on the new
server, but the backup with 10gb of media did not.



Em qua., 2 de fev. de 2022 21:38, ivellios1988 ***@***.***>
escreveu:
